docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,162 @@
+.. _acsincome-data:
+ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
+different algorithmic fairness interventions. ACSIncome offers a few improvements,
+such as providing more datapoints (1,664,500 vs. 48,842) and more recent data (2018 vs. 1994).
+Further, the binary labels in the UCI Adult dataset indicate whether an indivdual earned 
+more than $50k US dollars in that year. Ding et al. show that the choice of threshold impacts
+the amount of fairness violation, so they allow users to define any threshold.
+
+Ding et al. compiled data from the American Community Survey (ACS) Public Use Microdata Sample (PUMS). 
+Note that this is a different source than the Annual Social and Economic Supplement (ASEC) 
+of the Current Population Survey (CPS) used to construct the original UCI Adult dataset.
+Ding et al. filtered the data such that ACSIncome only includes individuals above 16 years old 
+who worked at least 1 hour per week in the past year and had an income of at least $100.
+
+
+.. _acsincome_dataset_description:
+
+Dataset Description
+^^^^^^^^^^^^^^^^^^^
+Ding et al. provide data from 2014-2018 for all 50 states and Puerto Rico.
+We uploaded the 2018 data to OpenML, which fairlearn can access.
```

Comment:
Anyone can access it 🙂 Also, Fairlearn is capitalized unless we're in code `fairlearn`

Reply:
Updated this description